src/core/bootstrap/bootstrap.service.ts

```diff
@@ -292,12 +293,20 @@ export class BootstrapService {
         },
         hostID: DEFAULT_HOST_ORG_NAMEID,
       };
-      return this.accountService
-        .createAccount(spaceInput)
-        .then(account =>
-          this.accountAuthorizationService.applyAuthorizationPolicy(account)
-        )
-        .then(account => this.accountService.save(account));
+
+      let account = await this.accountService.createAccount(spaceInput);
```

Comment:
here you are passing `NAMEID` to the `hostID` field, which cause the code at `account.service.ts:102` to fail, because it`s trying to read a contributor specifically by UUID, but you are passing a NAMEID instead.

**In cases where you enforce a certain format to the data, you need to validate it's the proper format.**